epytope/Data/pssms/syfpeithi/mat/B_5802_10.py

```diff
@@ -0,0 +1 @@
+B_5802_10 = {0: {'A': 0, 'C': 0, 'D': 0, 'E': 0, 'F': 0, 'G': 0, 'H': 0, 'I': 0, 'K': 4, 'L': 0, 'M': 0, 'N': 0, 'P': 0, 'Q': 0, 'R': 4, 'S': 0, 'T': 0, 'V': 0, 'W': 0, 'Y': 0}, 1: {'A': 8, 'C': 0, 'D': 0, 'E': 0, 'F': 0, 'G': 0, 'H': 0, 'I': 0, 'K': 0, 'L': 0, 'M': 0, 'N': 0, 'P': 0, 'Q': 0, 'R': 0, 'S': 10, 'T': 10, 'V': 0, 'W': 0, 'Y': 0}, 2: {'A': 0, 'C': 0, 'D': 0, 'E': 0, 'F': 0, 'G': 0, 'H': 0, 'I': 1, 'K': 0, 'L': 2, 'M': 0, 'N': 0, 'P': 0, 'Q': 0, 'R': 0, 'S': 0, 'T': 0, 'V': 1, 'W': 0, 'Y': 0}, 3: {'A': 0, 'C': 0, 'D': 1, 'E': 1, 'F': -1, 'G': 0, 'H': 0, 'I': 0, 'K': 0, 'L': 0, 'M': 0, 'N': 0, 'P': 2, 'Q': 0, 'R': 0, 'S': 0, 'T': 0, 'V': 0, 'W': -1, 'Y': -1}, 4: {'A': 1, 'C': 0, 'D': 1, 'E': 0, 'F': 0, 'G': 1, 'H': 0, 'I': 0, 'K': 0, 'L': 0, 'M': 0, 'N': 0, 'P': 0, 'Q': 0, 'R': 0, 'S': 1, 'T': 0, 'V': 0, 'W': 0, 'Y': 0}, 5: {'A': 0, 'C': 0, 'D': 1, 'E': 0, 'F': 0, 'G': 0, 'H': 0, 'I': 0, 'K': 1, 'L': 0, 'M': 0, 'N': 0, 'P': 0, 'Q': 0, 'R': 0, 'S': 0, 'T': 0, 'V': 0, 'W': 0, 'Y': 0}, 6: {'A': 0, 'C': 0, 'D': 0, 'E': 0, 'F': 0, 'G': 0, 'H': 0, 'I': 0, 'K': 0, 'L': 2, 'M': 0, 'N': 0, 'P': 0, 'Q': 0, 'R': 0, 'S': 0, 'T': 0, 'V': 0, 'W': 0, 'Y': 0}, 7: {'A': 0, 'C': 0, 'D': 0, 'E': 0, 'F': 0, 'G': 0, 'H': 0, 'I': 0, 'K': 0, 'L': 1, 'M': 0, 'N': 0, 'P': 0, 'Q': 0, 'R': 2, 'S': 0, 'T': 0, 'V': 0, 'W': 0, 'Y': 0}, 8: {'A': 0, 'C': 0, 'D': 0, 'E': 0, 'F': 0, 'G': 0, 'H': 0, 'I': 0, 'K': 1, 'L': 1, 'M': 0, 'N': 0, 'P': 0, 'Q': 0, 'R': 0, 'S': 0, 'T': 1, 'V': 1, 'W': 0, 'Y': 0}, 9: {'A': 0, 'C': 0, 'D': 0, 'E': 0, 'F': 10, 'G': 0, 'H': 0, 'I': 10, 'K': 0, 'L': 10, 'M': 0, 'N': 0, 'P': 0, 'Q': 0, 'R': 0, 'S': 0, 'T': 0, 'V': 0, 'W': 0, 'Y': 0}}
```

Comment:
Just noticed that some contain `X`, some don't. Does it make sense that it's not there in some cases?

Reply:
Good catch! I just checked the file I derived the matrices. Randomly some of them contain Xs and some not. But they Xs are mandatory as far as I know.